changelog.d/20211117_174041_rudyardrichter_config_get_app_url.rst

```diff
@@ -0,0 +1,10 @@
+..
+.. A new scriv changelog fragment
+..
+.. Add one or more items to the list below describing the change in clear, concise terms.
+..
+.. Leave the ":pr:`...`" text alone. When you open a pull request, GitHub Actions will
+.. automatically replace it when the PR is merged.
+..
+
+* Tweak `get_service_url` to be able to return correct webapp URLs (:pr:`NUMBER`)
```

Comment:
```suggestion
* Tweak ``get_service_url`` to be able to return correct webapp URLs (:pr:`NUMBER`)
```

The RST use of double-backticks is a frequent source of annoyance for me -- I make this mistake constantly, even after many years of working with sphinx.